src/common/types.ts

```diff
@@ -251,6 +264,24 @@ export type SelectedPhaseGroup = {
   winnersTargetPhaseId: number | null;
 };
 
+export type SelectedParryggEvent = {
+  id: string;
+  name: string;
+  hasSiblings: boolean;
+};
+
+export type SelectedParryggPhase = {
+  id: string;
+  name: string;
+  hasSiblings: boolean;
+};
+
+export type SelectedParryggBracket = {
+  id: string;
+  name: string;
+  hasSiblings: boolean;
+};
```

Comment:
use existing `SelectedEvent` `SelectedPhase` and `SelectedPhaseGroup` (same reasoning as `Context` below)